webpack.config.js

```diff
@@ -0,0 +1,97 @@
+const path = require('path'),
+  webpack = require('webpack'),
```

Comment:
Meh, looks bad, better separate it to individual `const`s
+ Use TerserWebpackPlugin